src/modules/lua.c

```diff
@@ -57,6 +59,8 @@ static mtev_hash_table mtev_lua_states;
 static pthread_mutex_t mtev_lua_states_lock = PTHREAD_MUTEX_INITIALIZER;
 static mtev_hash_table mtev_coros;
 static pthread_mutex_t coro_lock = PTHREAD_MUTEX_INITIALIZER;
+static stats_ns_t *lua_stats_ns;
+static stats_handle_t *gc_total, *gc_full, *gc_latency;
```

Comment:
We could also report `LUA_GCCOUNT` here fairly easily.

Reply:
it's a bit challenging to do it across all the VMs